.github/workflows/build-and-push-assets.yml

```
@@ -43,6 +43,11 @@ on:
         default: './assets'
         required: false
         type: string
+      MODE:
+        description: Define the mode for building either "prod" or "dev" assets.
+        default: ''
+        required: false
+        type: string
```

As this belongs to `COMPILE_SCRIPT_*`, let's move it above `ASSETS_TARGET_PATHS`.
```suggestion
      MODE:
        description: Mode for compiling assets (`prod` or `dev`)
        default: ''
        required: false
        type: string
```